diff --git a/CMake/cuda_config.cmake b/CMake/cuda_config.cmake
index d03a0f53..747404da 100644
--- a/CMake/cuda_config.cmake
+++ b/CMake/cuda_config.cmake
@@ -12,4 +12,4 @@ message(STATUS "CUDA_LIBRARIES: ${CUDA_INCLUDE_DIRS} ${ALL_CUDA_LIBS}")
 
 set(CUDA_PROPAGATE_HOST_FLAGS OFF)
 set(CUDA_SEPARABLE_COMPILATION ON)
-set(CUDA_NVCC_FLAGS "${CUDA_NVCC_FLAGS}; -O3 -gencode arch=compute_53,code=sm_53 -gencode arch=compute_62,code=sm_62;")
+set(CUDA_NVCC_FLAGS "${CUDA_NVCC_FLAGS}; -O3 -gencode arch=compute_72,code=sm_72;")
